### DISCUSSION TITLE
Which interface do you use to connect to the display?

### DISCUSSION THREAD

**Discussion by VasilKalchev** [VasilKalchev/LiquidMenu#91] [Polls]
.

**Comment by joe-ave** [comment]
I have an 20x4 lcd display that I use in 8 bit parallel mode due to that I sometimes have emi problems not fully sorted out yet caused by relays.
The emi problems causes the display to go nuts sometimes, but calling lcd.begin when switching menu brings it back again.
This only worked in 8 bit mode.
  lcd.begin(20, 4);
  lcd.clear();
  menu.update();

Maybe I add this rescue code later on after a relay has been switched, then i do not need to change menu to get it back.

//Jonas
